pom.xml

```diff
@@ -129,8 +129,8 @@
 				<artifactId>maven-compiler-plugin</artifactId>
 				<version>3.8.1</version>
 				<configuration>
-					<source>1.7</source>
-					<target>1.7</target>
+					<source>1.8</source>
```

Comment:
I'm not sure if updating it to 1.8 will cause compatibility problems for some users of 1.7. I think we decided to update it to 1.8, which should be done when a major version is released.